.github/workflows/workflow.yml

```diff
@@ -178,13 +179,40 @@ jobs:
         echo "${{ secrets.DOCKER_PRIVATE_KEY }}" > ~/.ssh/id
         echo "${{ secrets.DOCKER_KNOWN_HOSTS }}" > ~/.ssh/known_hosts
         ssh linkerd-docker docker version
-    - name: Docker build
+    - name: Build docker images (host)
       if: github.event_name != 'pull_request' || !github.event.pull_request.head.repo.fork
       env:
         DOCKER_HOST: ssh://linkerd-docker
-      run: |
-        export PATH="`pwd`/bin:$PATH"
-        DOCKER_TRACE=1 bin/docker-build
+      run: DOCKER_TRACE=1 bin/docker-build
+    - name: Build docker images (GitHub)
+      if: github.event.pull_request.head.repo.fork
+      run: DOCKER_TRACE=1:LINKERD_LOCAL_BUILD_CLI=1 bin/docker-build
```

Comment:
From my testing, in linux env vars declarations should be separated by space, not colon, so `LINKERD_LOCAL_BUILD_CLI` won't be set and this will build the CLI inside docker. BUT if you fix this, the `bin/docker cp` command below will fail.

Reply:
Yep good call this was easy to consolidate into a single `Build docker images` step.

Ah right `github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork` makes sense to ensure this path only happens for Pull Requests only.